<launch>
	<node pkg="facial_puppetry" name="dlib_core_node" type="dlib_core_node">
		<param name = "path" value= "$(find facial_puppetry)/data/shape_predictor_68_face_landmarks.dat"/>
	</node>
	<!--node pkg="facial_puppetry" name="pupeteery_core" type="pupeteery_core.py" output="screen" /-->
	<!--remap the topics if using multiple cameras -->
	<remap from="input_rgb_image" to="camera/image_raw" />
	<!--<remap from="/usb_cam/image_raw" to="/usb_cam_node/image_raw"/>-->
	<include file = "$(find usb_cam)/launch/usb_cam-test.launch"/>
	<include file = "$(find gui)/launch/gui.launch"/>
</launch>
